### ISSUE TITLE
Error update 2022.4.1

### ISSUE THREAD

**Issue by Mick51** [home-assistant/supervisor#3561]
### Describe the issue you are experiencing

```
22-04-11 22:48:05 ERROR (MainThread) [supervisor.store.git] Can't update https://github.com/home-assistant/addons repo: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: expected flush after ref listing'.
22-04-11 22:48:05 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.CORRUPT_REPOSITORY - ContextType.STORE / core
22-04-11 22:48:05 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_RESET - ContextType.STORE / core
22-04-11 22:48:12 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-21102' coro=<Repository.update() done, defined at /usr/src/supervisor/supervisor/store/repository.py:106> exception=StoreGitError()>
```

### What is the used version of the Supervisor?

2022.4.0

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### What is the version of your installed operating system?

7.6

### What version of Home Assistant Core is installed?

2022.4.2

### Steps to reproduce the issue

1. Start update
2.
3.
4.
...

### Anything in the Supervisor logs that might be useful for us?

```txt
22-04-11 22:48:05 ERROR (MainThread) [supervisor.store.git] Can't update https://github.com/home-assistant/addons repo: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: expected flush after ref listing'.
22-04-11 22:48:05 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.CORRUPT_REPOSITORY - ContextType.STORE / core
22-04-11 22:48:05 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_RESET - ContextType.STORE / core
22-04-11 22:48:12 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-21102' coro=<Repository.update() done, defined at /usr/src/supervisor/supervisor/store/repository.py:106> exception=StoreGitError()>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/git.py", line 155, in pull
    await self.sys_run_in_executor(
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 886, in fetch
    res = self._get_fetch_info_from_stderr(proc, progress,
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 750, in _get_fetch_info_from_stderr
    proc.wait(stderr=stderr_text)
  File "/usr/local/lib/python3.9/site-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: expected flush after ref listing'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/repository.py", line 110, in update
    await self.git.pull()
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 108, in wrapper
    raise err
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 106, in wrapper
    return await self._method(*args, **kwargs)
  File "/usr/src/supervisor/supervisor/store/git.py", line 195, in pull
    raise StoreGitError() from err
supervisor.exceptions.StoreGitError
```

**Comment by Mick51** [comment]
`22-04-11 23:10:55 WARNING (MainThread) [supervisor.host.network] Requested to update interface eth0 which does not exist or is disabled.`

**Comment by tescophil** [comment]
Also getting this issue of m HA core-2022.4.3, supervisor-2022.04.0

```
22-04-14 07:43:09 ERROR (MainThread) [supervisor.store.git] Can't update https://github.com/home-assistant/addons repo: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'.
22-04-14 07:43:09 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.CORRUPT_REPOSITORY - ContextType.STORE / core
22-04-14 07:43:09 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_RESET - ContextType.STORE / core
22-04-14 07:43:09 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-56153' coro=<Repository.update() done, defined at /usr/src/supervisor/supervisor/store/repository.py:106> exception=StoreGitError()>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/git.py", line 155, in pull
    await self.sys_run_in_executor(
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 886, in fetch
    res = self._get_fetch_info_from_stderr(proc, progress,
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 750, in _get_fetch_info_from_stderr
    proc.wait(stderr=stderr_text)
  File "/usr/local/lib/python3.9/site-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/repository.py", line 110, in update
    await self.git.pull()
  File 
"/usr/src/supervisor/supervisor/jobs/decorator.py", line 108, in wrapper
    raise err
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 106, in wrapper
    return await self._method(*args, **kwargs)
  File "/usr/src/supervisor/supervisor/store/git.py", line 195, in pull
    raise StoreGitError() from err
supervisor.exceptions.StoreGitError
```

**Comment by EDelsman** [comment]
Seeing the exact same behaviour (both eth0 warning and git update error)  on my Home Assistant running on pi4b, core-2022.4.3, supervisor-2022.04.0, home assistant OS 7.6 production.

Originally thought I caused this by installing Studio Code Server from HACS and specifying my github name and email, but now I'm not so sure anymore.

**Comment by tescophil** [comment]
> Originally thought I caused this by installing Studio Code Server from HACS and specifying my github name and email, but now I'm not so sure anymore.

I don't have HACS or Studio Code Server installed

**Comment by canedje** [comment]
Same problem here running a VM under Unraid at aNUC:
```
22-04-14 19:06:57 ERROR (MainThread) [supervisor.store.git] Can't update https://github.com/home-assistant/addons repo: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'.
22-04-14 19:06:57 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.CORRUPT_REPOSITORY - ContextType.STORE / core
22-04-14 19:06:57 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_RESET - ContextType.STORE / core
22-04-14 19:06:58 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-1729367' coro=<Repository.update() done, defined at /usr/src/supervisor/supervisor/store/repository.py:106> exception=StoreGitError()>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/git.py", line 155, in pull
    await self.sys_run_in_executor(
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 886, in fetch
    res = self._get_fetch_info_from_stderr(proc, progress,
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 750, in _get_fetch_info_from_stderr
    proc.wait(stderr=stderr_text)
  File "/usr/local/lib/python3.9/site-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/repository.py", line 110, in update
    await self.git.pull()
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 108, in wrapper
    raise err
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 106, in wrapper
    return await self._method(*args, **kwargs)
  File "/usr/src/supervisor/supervisor/store/git.py", line 195, in pull
    raise StoreGitError() from err
supervisor.exceptions.StoreGitError
```

**Comment by AndreyShpilevoy** [comment]
Same problem.

Generic x86-64 on NUC running core-2022.4.3, supervisor-2022.04.0, Home Assistant OS 7.6:

`22-04-15 00:10:02 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
22-04-15 00:10:02 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/zigbee2mqtt/hassio-zigbee2mqtt repository
22-04-15 00:10:02 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
22-04-15 00:10:02 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
22-04-15 00:10:02 ERROR (MainThread) [supervisor.store.git] Can't update https://github.com/home-assistant/addons repo: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'.
22-04-15 00:10:03 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-90276' coro=<Repository.update() done, defined at /usr/src/supervisor/supervisor/store/repository.py:106> exception=StoreGitError()>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/git.py", line 155, in pull
    await self.sys_run_in_executor(
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 886, in fetch
    res = self._get_fetch_info_from_stderr(proc, progress,
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 750, in _get_fetch_info_from_stderr
    proc.wait(stderr=stderr_text)
  File "/usr/local/lib/python3.9/site-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/repository.py", line 110, in update
    await self.git.pull()
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 108, in wrapper
    raise err
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 106, in wrapper
    return await self._method(*args, **kwargs)
  File "/usr/src/supervisor/supervisor/store/git.py", line 195, in pull
    raise StoreGitError() from err
supervisor.exceptions.StoreGitError`

**Comment by mdegat01** [comment]
Let's focus this isssue on the `StoreGitError` + Autofix exception that everyone is sharing. I realized I am also seeing this in my system too so something is going on.

My suspicion is that its related to #3567 . It appears the `hassio` component is forcing supervisor to do a full update check every 5 minutes. Which means every HA user with supervisor is cloning the core addons repo every 5 minutes. I'm guessing github may not like that.

@Mick51 @EDelsman can one of you make a new issue for this error if you are still seeing it:
```
22-04-11 23:10:55 WARNING (MainThread) [supervisor.host.network] Requested to update interface eth0 which does not exist or is disabled.
```
I'm not seeing details about that in most of these logs and its a separate problem. Would like to track it separately.

**Comment by VikeDragon** [comment]
same problem in
https://github.com/home-assistant/supervisor/issues/3573#issue-1205289140

**Comment by mdegat01** [comment]
A PR has been opened in core to address #3567 . I'm going to recommend we wait to see if returning to checking for updates in every addon repository every few hours instead of every 5 minutes resolved this. If not then we'll revisit

**Comment by AdmiralStipe** [comment]
Tried the PR and it really solves the 5 min checks - but it seems that it does not prevent this issue. Exactly one hour after restart of HA (after implementing the change from PR) I got this in my log:

```
22-04-15 17:26:06 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
22-04-15 17:26:06 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.CORE
22-04-15 17:26:06 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
22-04-15 17:26:06 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
22-04-15 17:26:06 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
22-04-15 17:26:06 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.PLUGIN
22-04-15 17:26:07 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
22-04-15 17:26:07 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-04-15 17:26:07 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-04-15 17:26:07 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-04-15 17:26:07 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-04-15 17:26:07 INFO (MainThread) [supervisor.resolution.fixups.store_execute_reset] Reset corrupt Store: core
22-04-15 17:26:07 INFO (MainThread) [supervisor.store.git] Cloning add-on https://github.com/home-assistant/addons repository
22-04-15 17:26:08 INFO (MainThread) [supervisor.resolution.fixups.store_execute_remove] Remove invalid Store: core
22-04-15 17:26:08 ERROR (MainThread) [supervisor.store.repository] Can't remove built-in repositories!
22-04-15 17:26:08 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/yllibed/hassio repository
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/esphome/home-assistant-addon repository
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/Poeschl/Hassio-Addons repository
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/sabeechen/hassio-google-drive-backup repository
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/zigbee2mqtt/hassio-zigbee2mqtt repository
22-04-15 17:26:41 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
22-04-15 17:26:42 ERROR (MainThread) [supervisor.store.git] Can't update https://github.com/hassio-addons/repository repo: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'.
22-04-15 17:26:42 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.CORRUPT_REPOSITORY - ContextType.STORE / a0d7b954
22-04-15 17:26:42 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_RESET - ContextType.STORE / a0d7b954
22-04-15 17:26:43 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-21885' coro=<Repository.update() done, defined at /usr/src/supervisor/supervisor/store/repository.py:106> exception=StoreGitError()>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/git.py", line 155, in pull
    await self.sys_run_in_executor(
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 886, in fetch
    res = self._get_fetch_info_from_stderr(proc, progress,
  File "/usr/local/lib/python3.9/site-packages/git/remote.py", line 750, in _get_fetch_info_from_stderr
    proc.wait(stderr=stderr_text)
  File "/usr/local/lib/python3.9/site-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git fetch -v --update-shallow --depth=1 origin
  stderr: 'fatal: Could not read from remote repository.'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/store/repository.py", line 110, in update
    await self.git.pull()
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 108, in wrapper
    raise err
  File "/usr/src/supervisor/supervisor/jobs/decorator.py", line 106, in wrapper
    return await self._method(*args, **kwargs)
  File "/usr/src/supervisor/supervisor/store/git.py", line 195, in pull
    raise StoreGitError() from err
supervisor.exceptions.StoreGitError
```
Now, maybe it's just the first attempt after changing files, that went wrong, I'll see how it goes in the next hour or two and will report back.

**Comment by AdmiralStipe** [comment]
Short update after two hours - stil not completely OK, but there's only one error left occuring exactly once per hour (first at 18:26 and the same one at 19:26): 

```
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.CORE
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.PLUGIN
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.fixups.store_execute_remove] Remove invalid Store: core
22-04-15 19:26:11 ERROR (MainThread) [supervisor.store.repository] Can't remove built-in repositories!
22-04-15 19:26:11 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
22-04-15 19:26:49 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
```

**Comment by mdegat01** [comment]
So @AdmiralStipe bear in mind here that while you may have changed your system to a much more stable update check schedule, every other supervisor user with core >= `2022.4` in the world is cloning that repository every 5 minutes. If I'm correct about github not liking the activity then one user changing to a slower schedule isn't going to solve that. We will need a new patch of HA before we can really reduce the activity on the repository and see if that's the cause.

**Comment by tescophil** [comment]
All I'm seeing now in the logs is this...

```
22-04-16 07:18:17 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-04-16 07:18:17 INFO (MainThread) [supervisor.resolution.fixups.store_execute_remove] Remove invalid Store: core
22-04-16 07:18:17 ERROR (MainThread) [supervisor.store.repository] Can't remove built-in repositories!
22-04-16 07:18:17 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
```

**Comment by martymarty004** [comment]
Maybe the issue will sort itself out after many users update to 2022.4.5 and fewer requests are sent to GitHub.

**Comment by mdegat01** [comment]
FYI 2022.4.5 out now with the mentioned fix for the hassio component

**Comment by mdegat01** [comment]
@martymarty004 yea that's my hope. Will need to give a few days for users to update. If still seeing it mid next week or so I'll dig in more and see if something else is going on

**Comment by keithr59** [comment]
Since updating to 2022.4.5 yesterday evening there have been no further errors and the add-on repositories have all loaded successfully. So far so good.

**Comment by tescophil** [comment]
After the 2022.4.5 upgrade I just get this error once every hour
```
[supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-04-17 10:19:05 INFO (MainThread) [supervisor.resolution.fixups.store_execute_remove] Remove invalid Store: core
22-04-17 10:19:05 ERROR (MainThread) [supervisor.store.repository] Can't remove built-in repositories!
22-04-17 10:19:05 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
```
No idea how to 'unmark' the core repo as corrupted....

**Comment by martymarty004** [comment]
> No idea how to 'unmark' the core repo as corrupted....

@tescophil  If I'm not mistaken, I think that you can use the `ha resolution issue dismiss` command through the terminal, but from what I understand, every time the error appears the repository is flagged as "corrupted" and System Autofix will try to fix it later.

Probably, when all the Resolution center issues will be "resolved", System Autofix will automatically stop.

![image](https://user-images.githubusercontent.com/61474670/163723696-4efc151b-17c2-484a-8885-9653f3230166.png)

**Comment by tescophil** [comment]
Can't find any documentation on this, and could not get any joy when I tried:
```
➜  ~ ha resolution info                                            checks:
- enabled: true
  slug: core_trust
- enabled: true
  slug: plugin_trust
- enabled: true
  slug: free_space
- enabled: true
  slug: addon_pwned
- enabled: true
  slug: supervisor_trust
- enabled: true
  slug: core_security
issues: []
suggestions:
- context: store
  reference: core
  type: execute_remove
  uuid: 48f89ce9116d4e80a6a99c7a0e992f2f
unhealthy: []
unsupported: []
➜  ~ ha resolution issue dismiss 48f89ce9116d4e80a6a99c7a0e992f2f  Error: The supplied UUID is not a valid issue
➜  ~ ha resolution issue dismiss core                              Error: The supplied UUID is not a valid issue
➜  ~
```

**Comment by martymarty004** [comment]
@tescophil try `ha resolution info`.

I think that command shows you the problems with the relative IDs.

**Comment by martymarty004** [comment]
I have never used this command before, maybe you have to enclose the UUID in quotations.

**Comment by martymarty004** [comment]
Ok, maybe it's because this is a _suggestion_ and not an _issue_.

Try if there is a similar command, like `ha resolution suggestion dismiss`

**Comment by canedje** [comment]
I do receive after update to 2022.4.5:
`22-04-18 12:58:03 ERROR (MainThread) [supervisor.store.repository] Can't remove built-in repositories!`

Also every hour

**Comment by martymarty004** [comment]
I used the SSH Terminal Addon to remove the suggestion for the removal of the repository.
I think this should stop the Resolution Center from trying to fix this non-existing issue.

![image](https://user-images.githubusercontent.com/61474670/163857828-5a85407c-d147-4c51-8ee6-044e3164abec.png)

**Comment by mdegat01** [comment]
Ok so reviewing my system this week it looks like this issue has completely gone away on my systems at this point. All issues related to this have the text `StoreGitError` in the log so here's a record of appearances of log entries with that text for the past week:
![Screen Shot 2022-04-19 at 2 37 49 PM](https://user-images.githubusercontent.com/2037026/164073093-c162e77e-2e99-4a04-9ac8-864e3170d133.png)

Note that the very last one at the end at 12:53 PM today was because I updated the adguard addon. So no URLs resolved briefly including the ones for all my addon repositories and then things resumed working correctly.

One thing I did do was re-add the community add-ons repository manually. It must've been automatically removed at some point during all the failures so I added <https://github.com/hassio-addons/repository> as a repository at some point. I should've tracked the time I did that but that might line up with when the errors stopped. So if anyone is seeing this that might be something to try. Though please let me know if that does fix the issue for you because I will want to look into how that can be auto-corrected better.

If you're still seeing the issue and re-adding the missing repositories also doesn't help then please @ me with the logs you're seeing. Might be something else going on.

**Comment by AdmiralStipe** [comment]
Not sure my issue is still directly this one (corrupt repository error is gone) and maybe it's more #3575 (these two seem to be very similar) but I still have hourly errors about removing built-in repos: 
```
22-04-19 20:26:53 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-04-19 20:26:53 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-04-19 20:26:53 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-04-19 20:26:53 INFO (MainThread) [supervisor.resolution.fixups.store_execute_remove] Remove invalid Store: core
22-04-19 20:26:53 ERROR (MainThread) [supervisor.store.repository] Can't remove built-in repositories!
22-04-19 20:26:53 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
```
I also tried removing and re-adding the hassio addons repo, but the error persists. 
Or is my error unrelated to this and I'm just too impatient (as I've been told to wait for a couple of days for things to settle)?

**Comment by mdegat01** [comment]
@AdmiralStipe have you restarted or repaired supervisor at all since this began? Looking at the logs of my production system I realized I updated my operating system on 4/16 around the time those logs appear to stop (which also restarts supervisor). This doesn't seem like something that should be necessary and so if it does fix it I'll have to figure out why. But seems at least worth a shot. 

First just try restarting or repairing supervisor, can do that in the UI or using one of these commands:
```
ha supervisor restart
```
or
```
ha supervisor repair
```
If that doesn't work then could try a host reboot. Can use the UI or
```
ha host reboot
```
Then at least we'll both have tried all the same steps on our machines. If that still doesn't fix it for you then we can keep digging.

**Comment by martymarty004** [comment]
To stop the Autofix error I simply told the Resolution Center to ignore this specific issue.

What I did was the following:

`ha resolution info`

You get the info about every problem that was detected by the System Evaluation (in this case it's a "suggestion").

`ha resolution suggestion dismiss UUID`

Using the UUID that you can find with the previous command, you tell the Resolution Center to dismiss the broken suggestion.

**Comment by AdmiralStipe** [comment]
@mdegat01, you are right. Restarting Supervisor didn't work at first, but then I had to restart my Raspberry because of an OS update anyway and after system restart the error is gone.

**Comment by mdegat01** [comment]
Ok so we have two things for people to try here if people are still seeing this:

1. Enter `ha resolution info` in the CLI. If you see `execute_remove` suggestions in the output then do `ha resolution suggestion dismiss <uuid from output>` and that should make it stop.
2. If the above doesn't work (or if you don't like messing with the CLI) try rebooting the host.

I have noted to take a look at 1 above for sure since I don't think it should ever suggest removing built-in repos, that doesn't make sense. That then hopefully can keep this from occurring again.

For others still facing this please try the above 2 steps and let me know if the error persists.

**Comment by Mick51** [comment]
The error persists no matter what I try.

![Screenshot_20220420-104805_Home Assistant](https://user-images.githubusercontent.com/40418405/164189538-881c5ac1-0dc7-4e07-8b5d-f515349e5a06.jpg)

**Comment by canedje** [comment]
> To stop the Autofix error I simply told the Resolution Center to ignore this specific issue.
> 
> What I did was the following:
> 
> `ha resolution info`
> 
> You get the info about every problem that was detected by the System Evaluation (in this case it's a "suggestion").
> 
> `ha resolution suggestion dismiss UUID`
> 
> Using the UUID that you can find with the previous command, you tell the Resolution Center to dismiss the broken suggestion.
> 
> ![image](https://user-images.githubusercontent.com/61474670/163857828-5a85407c-d147-4c51-8ee6-044e3164abec.png)

I don’t know what I did, but it worked.
No redlines anymore.
Thanks

**Comment by Mick51** [comment]
> > To stop the Autofix error I simply told the Resolution Center to ignore this specific issue.
> > What I did was the following:
> > `ha resolution info`
> > You get the info about every problem that was detected by the System Evaluation (in this case it's a "suggestion").
> > `ha resolution suggestion dismiss UUID`
> > Using the UUID that you can find with the previous command, you tell the Resolution Center to dismiss the broken suggestion.
> > ![image](https://user-images.githubusercontent.com/61474670/163857828-5a85407c-d147-4c51-8ee6-044e3164abec.png)
> 
> I don’t know what I did, but it worked. No redlines anymore. Thanks

This removed your deposit from the store as well, how are you going to put it back ?  Because you will no longer have some updates.

**Comment by rzv-trsc** [comment]
Rebooting the host did it for me.

**Comment by canedje** [comment]
> 

I don’t know
I didn’t realise this stopped updates

**Comment by martymarty004** [comment]
> This removed your deposit from the store as well, how are you going to put it back ?  Because you will no longer have some updates.

I don't think that doing that could break the updates. 
The system flagged the core repository as corrupted, but this is not the case.
By dismissing the suggestion you simply tell the system to do nothing about this issue.
If you apply the suggestion, it will probably remove the "corrupted" repo, which is what it is trying to do automatically.

**Comment by canedje** [comment]
Thanks for the answer.
We will at the next update😊

**Comment by Mick51** [comment]
I also did the same thing but I get a new error. How to solve it ?

`    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 139, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 205, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 65, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/addons.py", line 162, in list
    data_addons = [
  File "/usr/src/supervisor/supervisor/api/addons.py", line 164, in <listcomp>
    ATTR_NAME: addon.name,
  File "/usr/src/supervisor/supervisor/addons/model.py", line 135, in name
    return self.data[ATTR_NAME]
  File "/usr/src/supervisor/supervisor/store/addon.py", line 19, in data
    return self.sys_store.data.addons[self.slug]
KeyError: 'a0d7b954_unifi'
22-04-20 17:29:46 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 121, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/_compat.py", line 54, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 111, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 139, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 205, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 65, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/store.py", line 120, in store_info
    ATTR_ADDONS: [
  File "/usr/src/supervisor/supervisor/api/store.py", line 121, in <listcomp>
    self._generate_addon_information(self.sys_addons.store[addon])
  File "/usr/src/supervisor/supervisor/api/store.py", line 81, in _generate_addon_information
    ATTR_ADVANCED: addon.advanced,
  File "/usr/src/supervisor/supervisor/addons/model.py", line 222, in advanced
    return self.data[ATTR_ADVANCED]
  File "/usr/src/supervisor/supervisor/store/addon.py", line 19, in data
    return self.sys_store.data.addons[self.slug]
KeyError: '5c53de3b_esphome-dev'`

**Comment by Mick51** [comment]
> @Mick51 If you go to the [supervisor add-on store](https://my.home-assistant.io/redirect/supervisor_store/) do you see the community add-ons repo or esphome in the list of repositories? It looks like they may have been removed. There seems to be an issue with those to address where they can actually get removed and then have to be manually added back (unlike the core one which seems to self-correct but spews errors about it).
> 
> If you are missing those repos you can add each of them back with these buttons:
> 
> Community add-ons repo [![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://camo.githubusercontent.com/9e0c22cf56020b6ad057d06b87690b7a3ae14d8ada8b16a51abf6e8ab97df755/68747470733a2f2f6d792e686f6d652d617373697374616e742e696f2f6261646765732f73757065727669736f725f6164645f6164646f6e5f7265706f7369746f72792e737667)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository)
> 
> ESP Home [![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://camo.githubusercontent.com/9e0c22cf56020b6ad057d06b87690b7a3ae14d8ada8b16a51abf6e8ab97df755/68747470733a2f2f6d792e686f6d652d617373697374616e742e696f2f6261646765732f73757065727669736f725f6164645f6164646f6e5f7265706f7369746f72792e737667)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fesphome%2Fhome-assistant-addon)
> 
> I actually just noticed while doing this that I too was missing ESPHome. I don't currently use it so I wasn't aware but that one should always be there as well.

They are present and I cannot add them again or delete them. Why ?
![Capture d’écran 2022-04-20 175811](https://user-images.githubusercontent.com/40418405/164273358-d2b2579d-e686-44bb-b25b-c1d9114197de.jpg)

**Comment by Mick51** [comment]
Why all these new errors ?

![Capture d’écran 2022-04-20 181600](https://user-images.githubusercontent.com/40418405/164276463-888c02c1-824d-4514-af2b-a06533dadf61.jpg)

`22-04-20 17:57:06 ERROR (MainThread) [supervisor.store.git] Can't clone https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository repository: Cmd('git') failed due to: exit code(128)
  cmdline: git clone -v --recursive --depth=1 --shallow-submodules https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository /data/addons/git/5b119d61
  stderr: 'Cloning into '/data/addons/git/5b119d61'...
fatal: https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository/info/refs not valid: could not determine hash algorithm; is this a git repository?
'.
22-04-20 17:57:06 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.FATAL_ERROR - ContextType.STORE / 5b119d61
22-04-20 17:57:06 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_REMOVE - ContextType.STORE / 5b119d61
22-04-20 17:57:06 ERROR (MainThread) [supervisor.store] Can't load data from repository https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository`

**Comment by mdegat01** [comment]
@Mick51 Ok wait it looks like the addon repos are already in your list so no need to re-add them. They can't be removed because they are built-in repos so they are always supposed to be there.

> `5b119d61 22-04-20 17:57:06 ERROR (MainThread) [supervisor.store] Can't load data from repository https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository`

This looks like you right clicked on the button and copied the link then pasted it into the add repos box. You're just supposed to click the button and it takes you to the right place with the URL pre-filled. Right clicking the button and copying the button's link then pasting it into the add repo pop-up doesn't work because the button is not a link to an add-on repo, it's a link to add a particular addon repo to your HA. But doesn't matter anyway since it seems like you already have them.

Please remove `https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository` from your list of repos. After that if you haven't rebooted your host please try that then let me know if you still see errors in your supervisor log from `[supervisor.store.git]`

**Comment by Mick51** [comment]
> @Mick51 Ok wait it looks like the addon repos are already in your list so no need to re-add them. They can't be removed because they are built-in repos so they are always supposed to be there.
> 
> > `5b119d61 22-04-20 17:57:06 ERROR (MainThread) [supervisor.store] Can't load data from repository https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository`
> 
> This looks like you right clicked on the button and copied the link then pasted it into the add repos box. You're just supposed to click the button and it takes you to the right place with the URL pre-filled. Right clicking the button and copying the button's link then pasting it into the add repo pop-up doesn't work because the button is not a link to an add-on repo, it's a link to add a particular addon repo to your HA. But doesn't matter anyway since it seems like you already have them.
> 
> Please remove `https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository` from your list of repos. After that if you haven't rebooted your host please try that then let me know if you still see errors in your supervisor log from `[supervisor.store.git]`

How to remove it, it does not appear in the list of repositories.

**Comment by mdegat01** [comment]
@Mick51 ok your list of repos looks correct. If you still have that issue listed in `ha resolution info` you can dismiss it via the CLI or reboot the host to get rid of it. At this point do you still have new errors coming into your log from `[supervisor.store.git]` or `[[supervisor.store]` or only pre-existing ones?

**Comment by mdegat01** [comment]
Ingress errors are unrelated. I would guess that's due to a tab being left open or something and a failure on re-open but if it keeps appearing please submit it as a separate issue. For this issue just looking for ones that include `[supervisor.store.git]` or `[supervisor.store]`

**Comment by Mick51** [comment]
Still this problem

![Capture d’écran 2022-04-20 195731](https://user-images.githubusercontent.com/40418405/164293574-f99acab6-2cf3-447a-adbc-4a764124a171.jpg)

**Comment by mdegat01** [comment]
So from your posts I believe that's the one related to this error:
```
22-04-20 17:57:06 ERROR (MainThread) [supervisor.store.git] Can't clone https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository repository: Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository /data/addons/git/5b119d61 stderr: 'Cloning into '/data/addons/git/5b119d61'... fatal: https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository/info/refs not valid: could not determine hash algorithm; is this a git repository? '. 22-04-20 17:57:06 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.FATAL_ERROR - ContextType.STORE / 5b119d61 22-04-20 17:57:06 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_REMOVE - ContextType.STORE / 5b119d61 22-04-20 17:57:06 ERROR (MainThread) [supervisor.store] Can't load data from repository https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository
```
That error occurred because this URL `https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository` was added as a repository which isn't valid.

You should just dismiss that. You can do that in one of two ways:

1. Enter `ha resolution issue dismiss <UUID>` in the CLI (copy and paste that value in UUID, I can't because its a screenshot)
2. Reboot the host

If you continue to see it after that then we'll dig in more because there's something else going on. But based on your log that is my understanding, it should just be dismissed now that the invalid repo is gone and it'll move on.

**Comment by canedje** [comment]
Update to 2022.4.6 did work.
Update to 2022.4.7 won't work.
It is making a backup but not updating.
Any ideas?

**Comment by rzv-trsc** [comment]
> Update to 2022.4.6 did work. Update to 2022.4.7 won't work. It is making a backup but not updating. Error showing up:
> 
> ```
> 22-04-25 12:21:11 ERROR (SyncWorker_2) [supervisor.docker.interface] Can't install ghcr.io/home-assistant/qemux86-64-homeassistant:2022.4.7: 500 Server Error for http+docker://localhost/v1.41/images/create?tag=2022.4.7&fromImage=ghcr.io%2Fhome-assistant%2Fqemux86-64-homeassistant&platform=linux%2Famd64: Internal Server Error ("Get "https://ghcr.io/v2/home-assistant/qemux86-64-homeassistant/manifests/sha256:c5a0c7951731279ba00936c9cb2b60ec17ca512be3f76238fae821d7eb41b749": Get "https://ghcr.io/token?scope=repository%3Ahome-assistant%2Fqemux86-64-homeassistant%3Apull&service=ghcr.io": net/http: request canceled (Client.Timeout exceeded while awaiting headers)")
> 22-04-25 12:21:11 WARNING (MainThread) [supervisor.homeassistant.core] Updating Home Assistant image failed
> ```
> 
> Any ideas?
> 
> EDIT: After restart VM and then restart core update worked

Had the exact same issue. 

Rebooting the host did it. (HAOS on RPi 4).
Now facing the following errors on boot/startup:
```
22-04-25 14:13:09 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 121, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/_compat.py", line 54, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 111, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 139, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 205, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 65, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/addons.py", line 162, in list
    data_addons = [
  File "/usr/src/supervisor/supervisor/api/addons.py", line 164, in <listcomp>
    ATTR_NAME: addon.name,
  File "/usr/src/supervisor/supervisor/addons/model.py", line 135, in name
    return self.data[ATTR_NAME]
  File "/usr/src/supervisor/supervisor/store/addon.py", line 19, in data
    return self.sys_store.data.addons[self.slug]
KeyError: 'a0d7b954_vscode'
22-04-25 14:13:09 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 121, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/_compat.py", line 54, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/sentry_sdk/integrations/aiohttp.py", line 111, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 139, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/middleware/security.py", line 205, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 65, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/store.py", line 120, in store_info
    ATTR_ADDONS: [
  File "/usr/src/supervisor/supervisor/api/store.py", line 121, in <listcomp>
    self._generate_addon_information(self.sys_addons.store[addon])
  File "/usr/src/supervisor/supervisor/api/store.py", line 81, in _generate_addon_information
    ATTR_ADVANCED: addon.advanced,
  File "/usr/src/supervisor/supervisor/addons/model.py", line 222, in advanced
    return self.data[ATTR_ADVANCED]
  File "/usr/src/supervisor/supervisor/store/addon.py", line 19, in data
    return self.sys_store.data.addons[self.slug]
KeyError: 'a0d7b954_vscode'
```

@canedje, facing the same errors by any chance?
Everything seems to work fine otherwise.

**Comment by canedje** [comment]
No I do not have errors anymore after succeeded update to 2022.4.7
